SpatialConnect/SpatialConnect.h

```diff
@@ -52,20 +51,140 @@
 
 @property(readonly) RACMulticastConnection *serviceEvents;
 
+/**
+ This singleton of SpatialConnect is shared across your app.
+
+ @return instance of SpatialConnect
+ */
 + (id)sharedInstance;
 
+/**
+ @discussion This starts all the services in the order they were added. Data,
+ Sensor, Config, and Auth are all started. Backend Service waits for the Config
```

Comment:
Can you be explicit about the service names here?  "SCBackendService waits for the SCConfigService..."